src/test/java/lambda/part1/exercise/Exercise1.java

```diff
@@ -0,0 +1,126 @@
+package lambda.part1.exercise;
+
+import com.google.common.base.Predicate;
+import com.google.common.collect.FluentIterable;
+import java.util.Comparator;
+import lambda.data.Person;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1 {
+
+    private Comparator<Person> personComparator = new Comparator<Person>() {
```

Comment:
I think, it's better to put this to the method, where it's used. Same thing to another ```private Comparator<Person>```.

Reply:
Ok, fixed